ocaml/idl/api_messages.ml

```diff
@@ -96,6 +96,7 @@ let v6_license_server_version_obsolete = addMessage "LICENSE_SERVER_VERSION_OBSO
 let pvs_proxy_no_cache_sr_available = addMessage "PVS_PROXY_NO_CACHE_SR_AVAILABLE" 3L (* No cache storage available for pvs site on the host *)
 let pvs_proxy_setup_failed = addMessage "PVS_PROXY_SETUP_FAILED" 3L (* Setting up pvs proxy rules or pvs proxy daemon initialisation failed internally *)
 let pvs_proxy_no_server_available = addMessage "PVS_PROXY_NO_SERVER_AVAILABLE" 3L (* No pvs server available for pvs site on host *)
+let pvs_proxy_cache_invalid_size = addMessage "PVS_PROXY_CACHE_INVALID_SIZE" 3L (* Cache storage VDI size exceeds available control domain memory *)
```

Comment:
I don't think "invalid size" is the right message here. The size that was chosen is "valid", but there is not enough space on the SR at the moment, possibly due to something that changed _after_ setting up the PVS cache.

Regarding the code comment: note that, while the main reason for adding this message was the behaviour of the memory SR (which is stored in dom0 memory), the error condition may occur for other SR types as well. It is therefore better to make the message/comment more generic, so that it applies to any SR type. A better message would say that there is not enough space for the cache on the cache SR.